src/main/java/com/coveros/selenified/element/check/Equals.java

```diff
@@ -314,18 +314,18 @@ default String checkText(int row, int col, String expectedText, double waitFor,
      * @return String: the actual value of the element. null will be returned if the element isn't present or an input
      */
     @SuppressWarnings("squid:S1168")
-    default String checkValue(String expectedValue, double waitFor, double timeTook) {
+    String checkValue(String expectedValue, double waitFor, double timeTook) {
         // record the action and check this is an input element
-        if (!isPresentInput(getElement().prettyOutputStart() + " is not an input on the page", waitFor)) {
+        if (!isPresentInput(this.element.prettyOutputStart() + " is not an input on the page", waitFor)) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " is not an input on the page" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)